build.gradle

```diff
@@ -26,6 +26,12 @@ plugins {
   id 'net.researchgate.release' version '2.7.0'
 }
 
+if (!JavaVersion.current().java11Compatible) {
+  throw new GradleException("Java 11 or later is required to build Pantheon.\n" +
+  "  Detected version ${JavaVersion.current()}")
+}
+
```

Comment:
Pantheon PR to move to Java 11 also added some jvm args to suppress some warnings guess we should do the same?

Maybe also enable error prone again?

Reply:
love your work

Reply:
Think I've caught them all.